maps/torch/items/summarydocuments.dm

```diff
@@ -1,393 +1,375 @@
+	desc = "A \"for official use only\" memorandum detailing an officer's options with regards to administrative action as referenced in the disposition of offenses. Options include: counseling, corrective training, administrative reprimands, bar to reenlistment, suspension of favorable personnel actions (flagging), MOS reclassification, revocation of security clearance, relief from duties, adverse NCOER/OER, removal from special status, rehabilitative transfer, administrative reduction, and administrative separations."
+
+obj/item/documents/generic/sol/realitybreak	
+    name = "Bluespace drive apparitions"	
+    desc = "An \"internal use only\" memo sent from Expeditionary Command, based on a document from Krri'gli Engineering. It states that any humanoid figures seen during a bluespace jump are incorporeal apparitions and no mind should be paid to them, even if they appear familiar. In addition, it notes the existence of a bluespace \"double\" that living crew may experience and strongly urges them not to touch said double, as tempting as it may be, citing many cases of irrecoverable catatonia in those who have."
 
-obj/item/documents/generic/sol/realitybreak
-    name = "Bluespace drive apparitions"
-    desc = "\"Internal Use ONLY\" memo, sent from Expeditionary Command, based on a document from Krri'gli Engineering. It states that any humanoid figures seen during a bluespace jump are incorporeal apparitions and no mind should be paid to them, even if they appear familiar. In addition, it notes the existence of a bluespace "double" that living crew may experience and strongly urges them not to touch said double, as tempting as it may be, citing many cases of catatonia in those who have."
-	
 /obj/item/documents/generic/sol/njprules
-	name = "nonjudicial punishments"
-	desc = "\"For Official Use Only\" memorandum detailing options with regards to nonjudicially punishing uniformed servicemembers. Such punishments include admonishment/reprimand, forfeiture of pay, restriction, extra duty, reduction in rank, and correctional custody if the installation has an approved facility. Amount of punishment depends on rank of accused and rank of commander."
+	name = "non-judicial punishments"
+	desc = "A \"for official use only\" memorandum detailing options with regards to non-judicially punishing uniformed servicemembers. Such punishments include: admonishment/reprimand, forfeiture of pay, restriction, extra duty, reduction in rank, and correctional custody if the installation has an approved facility. The punishments available and their severity depend on the rank of the accused and the imposing officer."
```

Comment:
> and correctional custody if the installation has an approved facility.

Is not applicable, as that would be a Judicial Punishment.

A reword of this highlighted sentence to represent the ["Creative Punishments"](https://wiki.baystation12.net/Sol_Code_of_Uniform_Justice#Creative_punishments) might be applicable.